experiments/generic/node_memory_hog/node_memory_hog_ansible_logic.yml

```diff
@@ -0,0 +1,160 @@
+          vars:
+            stage: "PreChaosCheck"
+            exp_pod_name: "{{ chaos_pod_name }}"
+            engine_ns: "{{ a_ns }}"
+            message: "AUT is Running successfully"
+          when: "c_engine is defined and c_engine != ''"
+
+         ## READY TO HOG THE NODE Memory
+        - name: Ready to hog the node memory
+          include_tasks: "{{ c_util }}"
+
+        ## POST-CHAOS APPLICATION LIVENESS CHECK
+        - block:
+
+            - name: Verify AUT liveness post fault-injection
+              include_tasks: "/utils/common/status_app_pod.yml"
+              vars:
+                app_ns: "{{ a_ns }}"
+                app_label: "{{ a_label }}" 
+                delay: 2
+                retries: 90
+
+            ## AUXILIARY APPLICATION HEALTH CHECK STATUS
+            - block:
+
+                - include: /utils/common/status_app_pod.yml
+                  vars:
+                    app_ns: "{{ item.split(':')[0] }}"
+                    app_label: "{{ item.split(':')[1] }}"
+                    delay: 2
+                    retries: 90
+                  with_items:
+                    - "{{ auxiliary_appinfo_list }}"
+              
+              when: auxiliary_appinfo is defined and auxiliary_appinfo != ''
+
+          when: node_status is defined and node_status.stdout == 'Ready'
+          
+        - block:
+
+            - name: Verify AUT liveness post fault-injection
+              include_tasks: "/utils/common/status_app_pod.yml"
+              vars:
+                app_ns: "{{ a_ns }}"
+                app_label: "{{ a_label }}" 
+                delay: 2
+                retries: 90
+              ignore_errors: yes
+
+            ## AUXILIARY APPLICATION HEALTH CHECK STATUS
+            - block:
+
+                - include: /utils/common/status_app_pod.yml
+                  vars:
+                    app_ns: "{{ item.split(':')[0] }}"
+                    app_label: "{{ item.split(':')[1] }}"
+                    delay: 2
+                    retries: 90
+                  with_items:
+                    - "{{ auxiliary_appinfo_list }}"
+                  ignore_errors: yes
+
+              when: auxiliary_appinfo is defined and auxiliary_appinfo != ''
+
+          when: node_status is defined and node_status.stdout != 'Ready'
+
+        ## RECORD EVENT FOR POST-CHAOS CHECK
+        - include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
+          vars:
+            stage: "PostChaosCheck"
+            exp_pod_name: "{{ chaos_pod_name }}"
+            engine_ns: "{{ a_ns }}"
+            message: "AUT is Running successfully"
+          when: "c_engine is defined and c_engine != ''"
+
+        - set_fact:
+            flag: "Pass"
+
+      rescue:
+         
+        - set_fact:
+            flag: "Fail"
+    
+      always:
+
+        ## RECORD END-OF-TEST IN LITMUSCHAOS RESULT CR
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
```

Comment:
Can you also include the utils file to update the summary? pls refer to any other experiment